pytest.ini

```diff
@@ -12,4 +12,8 @@ env =
   TWILIO_NUMBER=5551112222
   TWITTER_CONSUMER_KEY=twitter-consumer-key
   TWITTER_CONSUMER_SECRET=twitter-consumer-secret
+  ORIGIN_SIGNING_KEY=0x0000000000000000000000000000000000000000000000000000000000000001
```

Comment:
Don't need this

Reply:
alright removed both of those references.